overview/intro.md

```diff
@@ -2,67 +2,113 @@
 layout: page_v2
 title: What is Prebid?
 description: An overview of Prebid org and products
-pid: 0
-is_top_nav: yeah
-top_nav_section: overview
-nav_section: intro
 sidebarType: 0
 ---
+# What is Prebid?
+{:.no_toc}
 
+* TOC
+{:toc}
 
+## Prebid Products, Community and Organization
```

Comment:
I'd title this section "Overview", then have an opening sentence along the lines of "Prebid is more than a product; it's a product suite, a community, and an organization." Then precede each bullet with, in bold, what piece you're describing: "Product" "Community" and "Organization"

Reply:
Changed, give it another look over to see if works better.